CONTRIBUTING.md

```diff
@@ -69,9 +69,8 @@ issue, provide us with:
 
 ### `rustfmt` / `cargo fmt`
 
-We use `nightly` channel for `rustfmt` so please set the appropriate setting your editor/IDE for that.
-
-For rust-analyzer, you can set `rustfmt.extraArgs = ['+nightly']`.
```

Comment:
I wouldn't remove this as I expect most contributors will be using rust-analyzer and will have a lot of formatting changes if their ra formats with the stable `rustfmt`. Maybe you can merge this and the "so please set the appropriate setting in your editor/IDE for that." with something like:

> We use the `nightly` release of `rustfmt`, so please set the appropriate setting in your editor/IDE for that. In the case of editors relying on rust-analyzer, you can set `rustfmt.extraArgs` to `[+nightly]`.